src/RestWrite.js

```diff
@@ -1104,6 +1102,29 @@ RestWrite.prototype.sanitizedData = function() {
   return Parse._decode(undefined, data);
 }
 
+// Returns an updated copy of the object
+RestWrite.prototype.buildUpdatedObject = function (extraData) {
+  var updatedObject = triggers.inflate(extraData, this.originalData);
```

Comment:
we use `const` for immutable variable (never reassigned) and `let` in place of vars as they behave more appropriately.